static/src/javascripts/projects/common/modules/experiments/tests/prebid-integration.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import { getTestVariantId } from 'common/modules/experiments/utils';
+import config from 'lib/config';
+
+export const prebidIntegration: ABTest = {
+    id: 'PrebidIntegration',
+    start: '2017-12-28',
+    expiry: '2018-04-02',
+    author: 'Richard Nguyen',
+    description:
+        'A test to verify that Prebid can support external demand for display ads',
+    audience: 0.0,
+    audienceOffset: 0,
+    successMeasure: 'Higher OMP (Open Market Place) yield',
```

Comment:
Is there any sensible `onComplete`/`onSuccess` we could put here to enrich the test data?

Reply:
Possibly the full ad load callback, but it's per slot, and lazy load. The best way to measure is to look at a report for the dfp line items for prebid, I reckon.